Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -627,6 +626,10 @@ open class MapView: UIView {
 
 extension MapView: DelegatingMapClientDelegate {
     internal func scheduleRepaint() {
+        guard let metalView = metalView, !metalView.bounds.isEmpty else {
+            return
+        }
+
```

Comment:
This might be unnecessary since the metal view is not supposed to have zero size. On the other hand this shouldn't not cause any problems so it might be an extra layer of protections against the crashes related to zero size.